extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/CockroachDialectConfig.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.hibernate.orm.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.configuration.TrimmedStringConverter;
+import io.smallrye.config.WithConverter;
+
+/**
+ * Configuration specific to the Hibernate ORM {@linkplain org.hibernate.dialect.CockroachDialect}
+ *
+ * @author Steve Ebersole
+ */
+@ConfigGroup
+public interface CockroachDialectConfig {
```

Comment:
So... maybe that's what you were getting at on Zulip when asking what should be exposed? Which _DBs_ are supported on Quarkus?

Because the list is here: https://quarkus.io/guides/datasource#configure-a-single-datasource

```
DB2: db2
Derby: derby
H2: h2
MariaDB: mariadb
Microsoft SQL Server: mssql
MySQL: mysql
Oracle: oracle
PostgreSQL: postgresql, pgsql or pg
```

Cockroach, HANA and Sybase are not officially supported in Quarkus -- as you may have noticed when you looked at `DatabaseKindDialectBuildItem`. I think it would be better to leave those configurable through `unsupported-properties` ATM, otherwise this will cause confusion as to which DBs are supported and which aren't...